Baikal/PostEffects/ML/denoiser.cpp

```diff
@@ -63,30 +60,30 @@ namespace Baikal
             }
 
             return std::make_unique<InferenceImpl>(model_path,
-                                                   gpu_memory_fraction,
-                                                   visible_devices,
-                                                   width,
-                                                   height,
+                                                   params.gpu_memory_fraction,
```

Comment:
I'd suggest to rename `MLDenoiserParams` to `InferenceParams` and pass them to InferenceImpl constructor.